synapse/federation/federation_server.py

```diff
@@ -112,10 +112,11 @@ def __init__(self, hs: "HomeServer"):
         # with FederationHandlerRegistry.
         hs.get_directory_handler()
 
-        self._federation_ratelimiter = hs.get_federation_ratelimiter()
```

Comment:
I wonder if we should just inline the rate-limiter that used to be shared with this into `TransportLayerServer` now...

Reply:
:man_shrugging: probably...